src/BlazorWasmSdk/Tasks/ComputeBlazorPublishAssets.cs

```diff
@@ -251,6 +268,23 @@ public override bool Execute()
             static bool IsDotNetWasm(string key) => string.Equals("dotnet.wasm", Path.GetFileName(key), StringComparison.Ordinal);
         }
 
+        private string GetItemHash(ITaskItem item)
+        {
+            var hash = item.GetMetadata("FileHash");
+            if (string.IsNullOrEmpty(hash))
+            {
+                // Some files that are part of the service worker manifest may not have their hashes previously
```

Comment:
Is this comment correct? This is only ever called on dotnet.js which isn't part of the manifest, no?

Reply:
You're right. This comment isn't accurate in this case. Copied the hashing/comment from another task in the pipeline which did deal with manifests. Removed the comment.

Reply:
Agree with Pranav, not sure what this comment means.

At this stage nothing has/needs a hash, we compute hashes when we are about to generate the blazor boot json manifest.